context/php/php-fpm.conf

```diff
@@ -0,0 +1,5 @@
+[global]
+pid = /var/run/php-fpm.pid
+error_log = /proc/self/fd/2
+log_level = notice
+include=/usr/local/etc/php-fpm.d/*.conf
```

Comment:
Please add (from original `/usr/local/etc/php-fpm.d/docker.conf` file, which we delete during build:
```
log_limit = 8192
```

Reply:
log_limit is available only for PHP 7.3
https://www.php.net/manual/en/install.fpm.configuration.php

For 7.1 and 7.2 we got:
[28-Mar-2019 15:12:57] ERROR: [/usr/local/etc/php-fpm.conf:5] unknown entry 'log_limit'